build.gradle

```diff
@@ -40,8 +40,10 @@ allprojects {
   }
 
   tasks.withType(KotlinCompile).configureEach {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+    kotlinOptions {
+      jvmTarget = "17"
+      apiVersion = "1.3"
+    }
   }
```

Comment:
i started just dropping these entirely, i'm not sure they actually help

Reply:
I don't know. I think we should use the toolchain feature starting with Kotlin 1.8 (and in sqldelight too). This will hopefully set everything automatically.

Reply:
Will do it tomorrow